src/sentinel.c

```diff
@@ -1846,8 +1846,7 @@ void loadSentinelConfigFromQueue(void) {
     return;
 
 loaderr:
-    fprintf(stderr, "\n*** FATAL CONFIG FILE ERROR (Redis %s) ***\n",
-        REDIS_VERSION);
```

Comment:
Printing out the version might by handy to figure out incompatibility easily for admins.

Reply:
I don't have a strong opinion about this. There are plenty of other places it would get printed higher up in the log.

Reply:
I decided to change it because that isn't true.

Reply:
@zuiderkwast FYI on this:
```
matolson@88665a372227 valkey % ./src/placeholderkv-server --whatever 17         

*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 2
>>> 'whatever "17"'
Bad directive or wrong number of arguments
```

It doesn't print the version, so it seems like it might be a shortcut to see what was happening.

Reply:
So is it useful to know the version in case you have many versions laying around?

I don't really care.